### ISSUE TITLE
Make AOT Ready

### ISSUE THREAD

**Issue by cubissimo** [nogorilla/angular-cc-library#2]
We Can't build prod version using AOT

**Comment by nogorilla** [comment]
@cubissimo I have a branch https://github.com/nogorilla/angular-cc-library/tree/aot-support that supports AOT compilation. Please review in your build.

**Comment by Captnwalker1** [comment]
Still isn't working for me:
```
ERROR in Error encountered resolving symbol values statically. Could not resolve angular-cc-library relative to .../src/app/app.module.ts., resolving symbol AppModule in .../src/app/app.module.ts, resolving symbol AppModule in .../src/app/app.module.ts

ERROR in ./src/main.ts
Module not found: Error: Can't resolve './$$_gendir/app/app.module.ngfactory' in '.../src'
 @ ./src/main.ts 4:0-74
 @ multi ./src/main.ts
```

In my package.json 
`"angular-cc-library": "nogorilla/angular-cc-library#aot-support",`

**Comment by matohawk** [comment]
Can you make a new release with the AOT support?
Because we have some issues with angular-cli 1.0.0 and your last release now.
We need a metadata in your release to fix this issue.

Thanks

**Comment by maitham1** [comment]
Hey man any chance of generating the metadata files neccessary?

**Comment by matohawk** [comment]
I have already clone your repos.
And make a  local package with the compile code in our project but the best is to use the remote package.

**Comment by nogorilla** [comment]
@matohawk are you saying that the `aot-support` branch does compile?

**Comment by matohawk** [comment]
Yes me made an aot build in our project last month. And we add it in our project like a node module local.

**Comment by nogorilla** [comment]
@matohawk but you used the `aot-support` branch?

**Comment by matohawk** [comment]
yes

**Comment by matohawk** [comment]
the latest tag cannot support angular 4 - we can't build so

**Comment by matohawk** [comment]
@nogorilla  we not used directly the AOT branch we the compilation of the AOT branch. And we made a local npm package in our project with this one.

**Comment by sinsunsan** [comment]
to be noted  the AOT branch have already been merged in master... but is some commits behind master... so better  use master instead the realease that date back february

**Comment by nogorilla** [comment]
closed via #15
